controllers/pullRequestsController.js

```diff
@@ -0,0 +1,47 @@
+const logger = require('../utils/logger')
+const fetch = require('../lib/fetch')
+// const getGithubId = require('../utils/getGithubId')
+const { fetchUser } = require('../models/users')
+
+const pullRequests = async (req, res) => {
+  try {
+    const BASE_URL = 'https://api.github.com'
+    const { user } = await fetchUser(req.params.id)
+    const url = `${BASE_URL}/search/issues?q=org:Real-Dev-Squad+author:${user.github_id}+type:pr`
```

Comment:
We will be using the github API functionality at multiple places in the code.
We better move the PRs fetch logic to a file, say `services/githubService.js` where we can add all other logic around fetching data from github.

Also the `getNames` function at https://github.com/Real-Dev-Squad/website-backend/pull/90/files#diff-3c24610be60a29530abbeabee1a488714820262747d9542ecd1f39e29d89139aR8 does not necessarily be defined in a separate file unless that's going to be used multiple times in the repo. It can be added in the service function.